cmd/manager/main.go

```diff
@@ -254,3 +272,58 @@ func initClusterStatusChecker(mgr ctrl.Manager) (clusterchecker.Checker, error)
 	}
 	return checker, nil
 }
+
+func initIndexers(mgr ctrl.Manager) (err error) {
+	// init node indexer for networks
+	if err = mgr.GetFieldIndexer().IndexField(context.TODO(), &networkingv1.Network{},
+		networking.IndexerFieldNode, func(obj client.Object) []string {
+			network, ok := obj.(*networkingv1.Network)
+			if !ok {
+				return nil
+			}
+
+			switch networkingv1.GetNetworkType(network) {
+			case networkingv1.NetworkTypeUnderlay:
+				return globalutils.DeepCopyStringSlice(network.Status.NodeList)
+			case networkingv1.NetworkTypeOverlay:
+				return []string{networking.OverlayNodeName}
+			default:
+				return nil
+			}
+		}); err != nil {
+		return err
+	}
+
+	// init network indexer for Subnets
+	if err = mgr.GetFieldIndexer().IndexField(context.TODO(), &networkingv1.Subnet{},
+		networking.IndexerFieldNetwork, func(obj client.Object) []string {
+			subnet, ok := obj.(*networkingv1.Subnet)
+			if !ok {
+				return nil
+			}
+
+			networkName := subnet.Spec.Network
+			if len(networkName) > 0 {
+				return []string{networkName}
+			}
+			return nil
+		}); err != nil {
+		return err
+	}
+
+	if feature.MultiClusterEnabled() {
```

Comment:
this indexer should be registered in remote cluster manager not local manager.

Reply:
fixed